lib/hexpm/emails/emails.ex

```diff
@@ -65,6 +65,15 @@ defmodule Hexpm.Emails do
     |> render("organization_invite.html")
   end
 
+  def package_published(user, package) do
+    email()
+    |> email_to(Enum.map(user.emails, fn email -> email.email end))
```

Comment:
this can be simply:

```elixir
|> email_to(user.emails)
```

but wdyt about sending the email to only user's primary email address?

```elixir
|> email_to(user)
```

(it wasn't clear to me how it works so for posterity, we have `defimpl Bamboo.Email, for: Hexpm.User` which picks the primary email address from the struct)

Reply:
Yeah, we should only send emails to the primary email.